src/cli/init_test.go

```diff
@@ -58,20 +60,47 @@ func TestInteractiveInit(t *testing.T) {
 		assert.NoError(t, err)
 		_, err = c.ExpectString("API host ")
 		assert.NoError(t, err)
-		time.Sleep(100 * time.Millisecond)
+		time.Sleep(20 * time.Millisecond)
 		_, err = c.SendLine(utils.TestApiHost())
 		assert.NoError(t, err)
 		_, err = c.ExpectString("Please enter Keboola Storage API token. The value will be hidden.")
 		assert.NoError(t, err)
 		_, err = c.ExpectString("API token ")
 		assert.NoError(t, err)
-		time.Sleep(100 * time.Millisecond)
+		time.Sleep(20 * time.Millisecond)
 		_, err = c.SendLine(utils.TestTokenMaster())
 		assert.NoError(t, err)
-		_, err = c.ExpectString("Allowed branches")
+		_, err = c.ExpectString("Allowed project's branches:")
+		assert.NoError(t, err)
+		time.Sleep(20 * time.Millisecond)
+		_, err = c.SendLine(Enter) // enter, first option "only main branch"
+		assert.NoError(t, err)
+		_, err = c.ExpectString(`Generate workflows files for GitHub Actions?`)
```

Comment:
Takto to vyzera v konzole:
![image](https://user-images.githubusercontent.com/19371734/127999955-a9fdc32d-85c3-43e5-8078-3581e3534a0c.png)